src/Dfe.PlanTech.Web/Program.cs

```diff
@@ -19,49 +13,28 @@
     options.EnableForHttps = true;
 });
 
-builder.Services.AddApplicationInsightsTelemetry();
-
-builder.Services.AddHttpContextAccessor();
-builder.Services.AddSingleton<ITelemetryInitializer, CustomRequestDimensionsTelemetryInitializer>();
-
-builder.Services.AddGoogleTagManager();
-builder.Services.AddCspConfiguration();
-// Add services to the container.
-
 builder.Services.AddControllersWithViews();
 
-builder.Services.AddGovUkFrontend();
-
 if (!builder.Environment.IsDevelopment())
 {
-    var keyVaultUri = $"https://{builder.Configuration["KeyVaultName"]}.vault.azure.net/";
-    var azureCredentials = new DefaultAzureCredential();
-
-    builder.Configuration.AddAzureKeyVault(new Uri(keyVaultUri), azureCredentials);
-
-    builder.Services.AddDbContext<DataProtectionDbContext>(options => options.UseSqlServer(builder.Configuration.GetConnectionString("Database")));
-    builder.Services.AddDataProtection()
-        .PersistKeysToDbContext<DataProtectionDbContext>()
-        .ProtectKeysWithAzureKeyVault(new Uri(keyVaultUri + "keys/dataprotection"), azureCredentials);
-
-    //Add overrides json for overwriting KV values for testing
-    if (builder.Environment.IsStaging())
-    {
-        builder.Configuration.AddJsonFile("overrides.json", true);
-    }
+    builder.Services.AddReleaseServices(builder.Configuration);
```

Comment:
idk if I said already but I really like this refactoring of program.cs, nice one